api/submission/fetch_submission_history_test.go

```diff
@@ -0,0 +1,122 @@
+package submission
+
+import (
+    "github.com/eriq-augustine/autograder/api/core"
+    "github.com/eriq-augustine/autograder/config"
+    "github.com/eriq-augustine/autograder/model"
+    "github.com/eriq-augustine/autograder/util"
+    "path/filepath"
+    "reflect"
+    "testing"
+)
+
+func TestFetchSubmissionHistory(test *testing.T) {
```

Comment:
I'm not a fan of this test.
The testing procedure and mechanisms are great (which I should hope so, since I wrote TestFetchSubmission).
But I don't think we need an almost duplicate test effectively testing disk.GetSubmissionContents.

Instead, try to make a test that tests some of the semantics specific to this endpoint.
E.g., testing the semantics I discussed in the comment about FoundSubmissions would be great.